app/controllers/application_controller/authentication_dependency.rb

```diff
@@ -5,7 +5,7 @@ class ApplicationController
 
   def current_scopes
     return [] unless logged_in?
-    session[:current_scopes] ||= current_user.github_client_scopes
+    session[:current_scopes] = current_user.github_client_scopes
```

Comment:
This seems like it is more correct. The `||=` here implies we were maybe trying to save an API call, but I don't think that's something we need / want.

@tarebyte can you double check this since I think you wrote it originally.

Reply:
This seems fine 👍

Reply:
Do you know why we're putting `current_scopes` into the session in the first place?